frontend/www/js/omegaup/groups.test.ts

```diff
@@ -72,29 +78,115 @@ describe('groups_utils', () => {
     });
   });
 
-  describe('cleanRecords', () => {
-    it('Should clean all null cells', () => {
-      const records = cleanRecords([
-        ['username-1', 'Developer Diana', 'MX', 'AGU', 'female', null],
-        ['username-2', null, 'MX', 'QUE', 'male', 'Best School'],
-      ]);
-
-      expect(records).toEqual([
-        ['username-1', 'Developer Diana', 'MX', 'AGU', 'female', undefined],
-        ['username-2', undefined, 'MX', 'QUE', 'male', 'Best School'],
-      ]);
+  describe('getCSVRecords', () => {
+    const fields = [
+      'username',
+      'name',
+      'country_id',
+      'state_id',
+      'gender',
+      'school_name',
+    ];
+    const records: (null | string | number)[][] = [
+      ['username-1', 'Developer Diana', 'MX', 'AGU', 'female', 'Best School'],
+      ['username-2', 'Dev Diane', 'MX', 'QUE', null, 'Best School'],
+    ];
+    const expectedFormattedRecords: { [key: string]: string }[] = [
+      {
+        username: 'username-1',
+        name: 'Developer Diana',
+        country_id: 'MX',
+        state_id: 'AGU',
+        gender: 'female',
+        school_name: 'Best School',
+      },
+      {
+        username: 'username-2',
+        name: 'Dev Diane',
+        country_id: 'MX',
+        state_id: 'QUE',
+        school_name: 'Best School',
+      },
+    ];
+
+    it('Should clean all null cells for optional fields', () => {
+      const formattedRecords = getCSVRecords<GroupCSVDatasetRecord>({
```

Comment:
```suggestion
      const formattedRecords = getCSVRecords<types.Identity>({
```

igual en los demás lugares donde se manda llamar `getCSVRecords`.

Reply:
Listo.